app.js

```diff
@@ -0,0 +1,262 @@
+        makeA.href = Tracker.cocktailOptionsRound2[i].link;
+        makeA.id = 'drinkTitle';
+        makeA.className = 'drinkTitle';
+        makeA.target = '_blank';
+        makeImg.src = Tracker.cocktailOptionsRound2[i].imagePath;
+        makeImg.className = 'cocktailImg';
+        makeA.appendChild(makeImg);
+        makeDiv.appendChild(makeA);
+        getRenderId.appendChild(makeDiv);
+      }
+    }
+    else {
+      console.log('fail');
+    }
+  },
+
+  // this method removes the last presented drink option to make room for the new ones
+  removeDrinkCard: function(){
+    var remove = document.getElementById("render");
+    while (remove.firstChild) {
+      remove.removeChild(remove.firstChild);
+    }
+  },
+//method to change images when different ingredients are selected
+  changeIngredientImageIng1: function(){
+    var getIngredientOneImageName = document.getElementById('ing1');
+    console.log(getIngredientOneImageName);
+    var getImagePath = 'url(assets/assetspng/' + ing1.options[ing1.selectedIndex].value + '.png)';
+    console.log(getImagePath);
+    document.getElementById('imageSpot1').style.backgroundImage = getImagePath;
+  },
+
+  changeIngredientImageIng2: function(){
+    var getIngredienttwoImageName = document.getElementById('ing2');
+    console.log(getIngredienttwoImageName);
+    var getImagePath = 'url(assets/assetspng/' + ing2.options[ing2.selectedIndex].value + '.png)';
+    console.log(getImagePath);
+    document.getElementById('imageSpot2').style.backgroundImage = getImagePath;
+  },
+
+
+
+  changeIngredientImageIng3: function(){
+    var getIngredientThreeImageName = document.getElementById('ing3');
+    console.log(getIngredientThreeImageName);
+    var getImagePath = 'url(assets/assetspng/' + ing3.options[ing3.selectedIndex].value + '.png)';
+    console.log(getImagePath);
+    document.getElementById('imageSpot3').style.backgroundImage = getImagePath;
+  },
+
+  //method to make selecters go back to default
+  ingredientsBackToDefault:function(){
+    ing1.options[0].selected="selected";
+    ing2.options[0].selected="selected";
+    ing3.options[0].selected="selected";
+  },
+
+  //method to change ingredients back to cubes
+  changeBackToCubes: function(){
+    var getIngredientOneImageName = document.getElementById('ing1');
+    var getIngredientTwoImageName = document.getElementById('ing2');
+    var getIngredientThreeImageName = document.getElementById('ing3');
+    var getImagePath = 'url(assets/SqIceCube.png)';
+    document.getElementById('imageSpot1').style.backgroundImage = getImagePath;
+    document.getElementById('imageSpot2').style.backgroundImage = getImagePath;
+    document.getElementById('imageSpot3').style.backgroundImage = getImagePath;
+  },
+};
+// This is where the submit button is being tracked and starts the functionality.
+window.addEventListener("load", Tracker.dynamicallyLoadedIngredients);
+ing1.addEventListener('change', Tracker.changeIngredientImageIng1);
```

Comment:
You can DRY these event listeners and handlers out as well by listening for a change event at a higher level, and then utilizing the event object to drill down into which of the select boxes changed.